ionic2/app/pages/map/map.page.html

```diff
@@ -10,4 +10,7 @@
   </button>
 
   <poke-poi-card #poiCard></poke-poi-card>
+
+  <button id="artificial-mob-button"
```

Comment:
How is triggering it through the console cumbersome? And even if it is, this functionality will only ever be used for testing purposes by developers. So it doesn't have to be easily accessible by everyone.